src/ex_getln.c

```diff
@@ -2716,6 +2720,8 @@ check_opt_wim(void)
 	    new_wim_flags[idx] |= WIM_LIST;
 	else if (i == 8 && STRNCMP(p, "lastused", 8) == 0)
 	    new_wim_flags[idx] |= WIM_BUFLASTUSED;
```

Comment:
Per the comment above, please keep this in sync with `p_wim_values` and add this to that array. You won't get auto-complete otherwise.